### PR TITLE
update trinity easyblock for latest version 2.8

### PR DESCRIPTION
**Pull request by pescobar**

### CODE REVIEW

File: easybuild/easyblocks/t/trinity.py [hunk 1]
```diff
@@ -319,11 +319,16 @@ def sanity_check_step(self):
         else:
             sep = '_r'
 
+        if version >= LooseVersion('2.8'):
```
Comment (boegel) [inline easybuild/easyblocks/t/trinity.py]: @pescobar After a quick test, I noticed that this breaks the installation of some old `Trinity` easyconfigs like `Trinity-2012-10-05-goolf-1.4.10.eb`:

```
eb Trinity-2012-10-05-goolf-1.4.10.eb -df --include-easyblocks trinity.py
...
== sanity checking...
  >> file 'trinityrnaseq_r2012-10-05/Inchworm/bin/inchworm' found: OK
  >> file 'trinityrnaseq_r2012-10-05/Chrysalis/bin/Chrysalis' found: FAILED
```

Maybe enhance this to:

```suggestion
if version >= LooseVersion('2.8') and version < LooseVersion('2000'):

**Comment by pescobar** [comment]
@boegel I used the new feature in github to directly commit your suggestion. Can you give it another try?

File: easybuild/easyblocks/t/trinity.py [hunk 2]
```diff
@@ -319,11 +319,16 @@ def sanity_check_step(self):
         else:
             sep = '_r'
 
+if version >= LooseVersion('2.8') and version < LooseVersion('2000'):
```
[the remark this answers is not in the document]
Comment (boegel) [reply easybuild/easyblocks/t/trinity.py]: @pescobar Indentation is all messed up now, that's my bad... Hopefully this fixes it.

```suggestion
        if version >= LooseVersion('2.8') and version < LooseVersion('2000'):

**Comment by damianam** [comment]
This lgtm. @boegel?

**Comment by boegel** [comment]
Thanks @pescobar and @ebirn!

**Pull request merged**